### ISSUE TITLE
Cannot render `align` environment (new feature of KaTeX 0.13)

### ISSUE THREAD

**Issue by hotarugali** [yzhang-gh/vscode-markdown#943]
## Problem
Katex supports `align` environment [https://katex.org/docs/support_table.html](https://katex.org/docs/support_table.html), but it seems that Markdown All In One doesn't support. When I write `align` environment in my vscode, Markdown All In One render error, just as follows:

![图片](https://user-images.githubusercontent.com/32862896/116959204-78cf6e80-accf-11eb-9274-7b064f06afdb.png)

Or are there some options needed to be set?
Thanks.

**Comment by yzhang-gh** [comment]
Thanks for the feedback.

I have updated the KaTeX module. Please check out the [dev build](https://github.com/yzhang-gh/vscode-markdown#latest-development-build).

**Comment by Lemmingh** [comment]
It is **strange and tedious** that we have to bump `markdown-it-katex` every time updating KaTeX.

We should relax the range [there](https://github.com/yzhang-gh/markdown-it-katex) and **control its version here**.

What about:

```json
{
  "dependencies": {
    "katex": "*"
  },
  "devDependencies": {
    "jest": "^26.6.3",
    "markdown-it": "12",
    "markdown-it-testgen": "^0.1.6"
  },
  "peerDependencies": {
    "katex": "0",
    "markdown-it": ">=6"
  }
}
```

**Comment by Lemmingh** [comment]
In an X-Range, `x` is optional. Thus, `0` is equivalent to `0.x.x`, and is interpreted as `>=0.0.0 <1.0.0`.

---

`dependencies` declares requirement.

`"katex": "*"` says the package relies on any KaTeX.

`peerDependencies` declares compatibility.

`"katex": "0"` says the package is expected to be compatible with all KaTeX `0.x.x` versions.

npm v6 and v7 process `peerDependencies` slightly differently. But it shouldn't affect us in a few years.

---

In this way, `markdown-it-katex` won't lock things tightly.

**Comment by yzhang-gh** [comment]
Thanks

**Comment by Firestar-Reimu** [comment]
Update 2021.11

still does not support align

![Screenshot_20211109_202630](https://user-images.githubusercontent.com/65082432/140924075-5bb60f2a-bd3c-497e-9a2c-a6ca49576acd.png)

**Comment by Lemmingh** [comment]
#966

Sorry for the inconvenience. Please try the dev build for now:
<https://github.com/yzhang-gh/vscode-markdown#latest-development-build>

**Comment by Firestar-Reimu** [comment]
OK, solved. Wish that it will release soon.

**Comment by yzhang-gh** [comment]
released in v3.4.2
